demo/callbacks.py

```
@@ -280,6 +295,178 @@ def _evaluation_unavailable() -> html.Div:
+    level_bg = level_colors.get(level, "#95a5a6")
+
+    per_category: dict[str, int] = defaultdict(int)
+    per_layer: dict[str, int] = defaultdict(int)
+    for f in classification.findings:
+        per_category[f.category.value] += 1
+        per_layer[f.source.split(".")[0]] += 1
+
+    category_rows = [
+        html.Tr(
+            [
+                html.Td(cat, style=cell_style), 
```

Ruff (0.15.10)

[error] Undefined name `cell_style`

(F821)